sources/packages/web/src/main.ts

```diff
@@ -47,5 +48,6 @@ AppConfigService.shared.init().then(() => {
     .component("FormioContainer", FormioContainer)
     .component("FooterButtons", FooterButtons)
     .component("TitleValue", TitleValue)
+    .component("HorizontalSeparator", HorizontalSeparator)
```

Comment:
👍

Reply:
I will delete this component. We can use the `v-divider` instead.

Reply:
Component removed.